buildSrc/build.gradle

```diff
@@ -15,13 +15,31 @@
  */
 plugins {
     id 'groovy'
+    id 'java-gradle-plugin'
     id 'idea'
     id 'eclipse'
 }
 
 sourceCompatibility = '1.7'
 targetCompatibility = '1.7'
 
+gradlePlugin {
+    plugins {
+        jsoup {
+            id = "jsoup"
+            implementationClass = "org.gradle.plugins.jsoup.JsoupPlugin"
+        }
+        buildTypes {
+            id = "org.gradle.build-types"
```

Comment:
Why not rename this to simply `build-types`?

Reply:
Makes sense, done ✅